src/main/java/org/assertj/core/api/recursive/comparison/FieldLocation.java

```diff
@@ -22,8 +22,6 @@
 public class FieldLocation implements Comparable<FieldLocation> {
```

Comment:
@joel-costigliola does this class really need to be `public`? (same question for all the non overridden methods)

Reply:
The intention was to model field location for the recursive comparison, I realized later that it would a pain for users to build instances of this class instead of passing `String`.

`FieldLocation` is still used in [RecursiveComparisonConfiguration.registerComparatorForField](https://www.javadoc.io/static/org.assertj/assertj-core/3.16.1/org/assertj/core/api/recursive/comparison/RecursiveComparisonConfiguration.html#registerComparatorForField(java.util.Comparator,org.assertj.core.api.recursive.comparison.FieldLocation)) so we can't make package private yet (although I have deprecated the method it in 3.17.0).